tests/data_sources/gsp/test_gsp_data_source.py

```diff
@@ -46,6 +41,13 @@ def test_gsp_pv_data_source_get_locations_for_batch():
     locations_x, locations_y = gsp.get_locations_for_batch(t0_datetimes=gsp.gsp_power.index[0:10])
 
     assert len(locations_x) == len(locations_y)
+    assert locations_x[0] > 90  # this makes sure it is not in lat/lon
```

Comment:
OSGB coordinates _can_ be less than or equal to 90 :)  I'm not sure of any perfect way to assert that a coordinate is OSGB or lat,lon.  Other than, maybe, checking that it's within a certain geospatial boundary (e.g. is it on land within the UK?)  But that's maybe overkill?!

Reply:
hmm, good point, I think if it was less than 90 it would be in the sea - and we shouldnt have a GSP centroid there, so I'm happy to keep it like this - perhaps Ill add a comment in the code?

Reply:
Sure, sounds good:  Leave the code as-it but add a comment to explain...